tools/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ foreach(src ${TOOLS})
 endforeach()
 
 add_executable(db_stress${ARTIFACT_SUFFIX} db_stress.cc db_stress_tool.cc)
-target_link_libraries(db_stress${ARTIFACT_SUFFIX} ${LIBS})
+target_link_libraries(db_stress${ARTIFACT_SUFFIX} ${ROCKSDB_LIB})
```

Comment:
could you squash this change into "Minor clean-up to library definitions in CMakeLists.txt"?

Reply:
My understanding from past PRs is that the RocksDB team prefers if pull requests aren't advanced by non-fast-forward commits.  The entire change will be squashed into a single commit when it lands anyway.